src/main/webapp/search_area.js

```diff
@@ -67,106 +51,49 @@ class SearchArea {
     this.form.appendChild(this.zipcodeSubmit);
 
     this.zipcodeFormArea.appendChild(this.form);
-    this.organizationSearchArea.appendChild(this.zipcodeFormArea);
+    this.searchArea.appendChild(this.zipcodeFormArea);
 
     this.filterTagArea = new FilterTagArea(this);
-    this.filterTagArea.filterEntry.filterEntryArea.addEventListener('onParamEntry', 
+    this.filterTagArea.filterEntry.filterEntryArea.addEventListener('onParamEntry',
       (e) => this.setUrlParamValue(e.detail.urlParamKey, e.detail.urlParamValue), true);
 
-    this.loadMoreButton = document.createElement("div");	
-    this.loadMoreButton.setAttribute("class", "load-more-button");	
-    this.loadMoreButton.textContent = "See More Results";	
-    this.loadMoreButton.addEventListener('click', () => this.handleOrganizations());
-
-    this.organizationListArea = document.createElement("div");
-    this.organizationListArea.setAttribute("id", "organization-list");
-
-    this.organizationPopupArea = document.createElement("div");
-    this.organizationPopupArea.setAttribute("id", "organization-popup-area");
-    this.organizationPopupArea.classList.add("hide-popup");
-
-    this.organizationSearchArea.appendChild(this.organizationListArea);
-    this.organizationSearchArea.appendChild(this.loadMoreButton);
-    this.searchArea.appendChild(this.organizationSearchArea);
-    this.searchArea.appendChild(this.organizationPopupArea);
-  }
-  
-  refreshOrganizationList() {
-    this.filterParams.set("cursor", "none");	
-    this.lastResultFound = false;	
-    this.loadMoreButton.classList.remove("hide-load-button");	
-    this.organizationObjectsList = [];	
-    this.organizationListArea.innerHTML = "";	
-    this.handleOrganizations();
-  }
-
-  async handleOrganizations() {
-    if (!this.lastResultFound) {
-      await this.getListOfOrganizations();
-      this.renderListOfOrganizations();
+    this.loadMoreButton = document.createElement("div");
+    this.loadMoreButton.setAttribute("class", "load-more-button");
+    this.loadMoreButton.textContent = "See More Results";
+    this.loadMoreButton.addEventListener('click', () => this.handleObjects());
+
+    this.listArea = document.createElement("div");
+    this.popupArea = document.createElement("div");
+    console.log(this.forOrganizations);
+    if (this.forOrganizations) {
+      this.listArea.setAttribute("id", "organization-list");
```

Comment:
can you call this 

search-result-list and then not have the if/else ? I think that means you can then remove this.forOrganizations